e2e/libs/persistent_volume_claim/base.py

```diff
@@ -0,0 +1,11 @@
+from abc import ABC, abstractmethod
+
+class Base(ABC):
```

Comment:
If pvc has only one possible implementation (which is crd), why do we still need this abstract class to define the interfaces to be implemented? Since there's no other one to extend and have its own implementation.